cccatalog-api/cccatalog/api/controllers/search_controller.py

```diff
@@ -24,6 +24,9 @@ def search(search_params, index, page_size, page=1) -> Response:
     :param index: The Elasticsearch index to search (e.g. 'image')
     :param page_size: The number of results to return per page.
     :param page: The results page number.
+    :param ip: The user's  hashed IP. Hashed IPs are used to anonymously but
```

Comment:
very minor nitpick, there are two spaces between "user's" and "hashed".

Reply:
Good catch! Fixed